src/common/Misc/Misc.cpp

```diff
@@ -121,8 +134,6 @@ void Trace(const char* pFormat, ...)
 //
 bool ReadFile(const char *name, char **data, size_t *size, bool isbinary)
 {
-    FILE *file;
-
     //Open file
     if (fopen_s(&file, name, isbinary ? "rb" : "r") != 0)
```

Comment:
This looks unintentional? `file` will be undefined below here, this can't be right.